src/renderer/components/Config/PrereqsView.tsx

```diff
@@ -91,6 +91,27 @@ const PrereqsView = ({ sx }: Props) => {
           .
         </Typography>
       )
+    } else if (status.id === 'ps1ExecutionPolicy') {
+      status.description = (
+        <Typography fontSize={14}>
+          <span style={{ fontSize: 14, opacity: 0.6 }}>
+            Check whether the execution policy is set to allow unsigned PowerShell scripts.
+          </span>
+          <a style={{ color: 'white' }} target="_blank" href={Endpoints.Urls.SET_EXECUTION_POLICY}>
+            Learn more
+          </a>
+          .
+          <br />
+          <br />
+          <br />
+          <span style={{ fontSize: 14, opacity: 0.6 }}>
+            Afterwards, if the execution policy is not set to allow unsigned PowerShell scripts, you can do so by
```

Comment:
- Add `Get-ExecutionPolicy` command in the message.
- Move learn more to the end.